src/main/java/seedu/nutrigoals/logic/parser/ParserUtil.java

```diff
@@ -126,6 +126,12 @@ public static Location parseLocation(String locationName) throws ParseException
         out.put("LT9", new Location("LT9", "1.2952456082872508, 103.77221840699619"));
         // https://goo.gl/maps/EEWy1fTuFcBkqxT97
         out.put("AS6", new Location("AS6", "1.2955038493756006, 103.77324977312634"));
+        // https://goo.gl/maps/JqvjteGB87DLP5eL7
+        out.put("IT", new Location("IT", "1.2975840550675737, 103.77251102348825"));
+        // https://goo.gl/maps/foRXfEUYG3v3QKDi8
+        out.put("TCOMS", new Location("TCOMS", "1.2935168791869454, 103.77690208272466"));
+        // https://goo.gl/maps/SMVTaR564Hoahr1B8
+        out.put("VENTUS", new Location("VENTUS", "1.295314453199568, 103.77019611100641"));
```

Comment:
Not sure if we can add these, but to address the feature flaw mentioned by the tester, could perhaps mention in the UG that more locations will be added in future releases~ (can refer to the food items section for the `find` command in the UG for an example)